db/migrate/20111228035029_create_nav_items.rb

```diff
@@ -1,4 +1,4 @@
-class CreateNavItems < ActiveRecord::Migration
+class CreateNavItems < ActiveRecord::Migration[5.0]
```

Comment:
Yeah, I didn't give too much thought about what should happen with these things. As we discussed, technically they should probably be `ActiveRecord::Migration[4.2]`, but they all seem to run just fine with `ActiveRecord::Migration[5.0]`, so I think we'll leave them. 

Any future migrations will generate with `[5.1]`.